recipe/bld.bat

```diff
@@ -27,9 +27,9 @@ if errorlevel 1 exit 1
     variant=release ^
     threading=multi ^
     link=static,shared ^
-    -j%CPU_COUNT% ^
-    -s ZLIB_INCLUDE="%LIBRARY_INC%" ^
-    -s ZLIB_LIBPATH="%LIBRARY_LIB%"
+    -j%CPU_COUNT%
+::  -s ZLIB_INCLUDE="%LIBRARY_INC%" ^
+::  -s ZLIB_LIBRARY_PATH="%LIBRARY_LIB%"
```

Comment:
It is. That's what came out of the [issue]( https://svn.boost.org/trac/boost/ticket/12505 ) @jschueller raised upstream. Honestly we should probably just drop these commented lines, but I think he was just trying to test if this would fix it or not.